pkg/logs/tekton_logging.go

```diff
@@ -46,14 +46,21 @@ func GetTektonPipelinesWithActivePipelineActivity(jxClient versioned.Interface,
 	}
 
 	// This is a temporary solution until we add the "context" label to PipelineActivities
-	labelsFilter = strings.Replace(labelsFilter, "repository=", "repo=", 1)
 	if context != "" {
 		labelsFilter = modifyFilterForPipelineRun(labelsFilter, context)
 	}
 	tektonPRs, _ := tektonClient.TektonV1alpha1().PipelineRuns(ns).List(metav1.ListOptions{
 		LabelSelector: labelsFilter,
 	})
 
+	// Handle the old "repo" tag as well for legacy purposes.
+	if len(tektonPRs.Items) == 0 {
+		labelsFilter = strings.Replace(labelsFilter, "repository=", "repo=", 1)
```

Comment:
It's going to have a "repository" tag instead of a "repo" one now? Nice.

Reply:
@hferentschik did this last week. =)

Reply:
Now one can find all resources related to a pipeline execution (including the pipeline activity) using the same label, something like:

```
for i in pipelineactivity pipeline pipelinestructure taskrun pipelinerun task pod; do 
   kubectl get "$i" --show-kind --no-headers --ignore-not-found -l repository=demo,build=<x>
  done
```